app/bundles/LeadBundle/Tests/Entity/ImportTest.php

```diff
@@ -192,7 +192,7 @@ public function testGetSpeedWhenRunTimeIsUnderOneSecond()
         $import->setInsertedCount(3);
         $import->end(false);
 
-        $this->assertSame(3, $import->getSpeed());
+        $this->assertSame(3.0, $import->getSpeed());
```

Comment:
Maybe this would do it?
```suggestion
        $this->assertSame(3, (int) $import->getSpeed());
```

Reply:
Somehow this seems to be working just fine on my local environment exactly the way it was before. I have reset the head and force pushed it. And I am pretty sure that the tests are passing now xD.